user-org-actor/src/main/java/org/sunbird/user/UserAttributeProcessorActor.java

```diff
@@ -133,7 +125,27 @@ public Response apply(Iterable<Object> futureResult) {
     }
 
     private Future<Object> saveAddress(Map<String, Object> userMap) {
+        if (CollectionUtils.isNotEmpty((List<Map<String, Object>>) userMap.get(JsonKey.ADDRESS))) {
+            Map<String, Object> entity = new HashMap<>();
+            entity.put(JsonKey.ADDRESS, userMap.get(JsonKey.ADDRESS));
+            entity.put(JsonKey.USER_ID, userMap.get(JsonKey.USER_ID));
+            return saveEntity(entity, UserActorOperations.CREATE_ADDRESS.getOperation());
+        }
         return null;
     }
 
+    private Future<Object> saveEntity(Map<String, Object> entity, String actorOperation) {
```

Comment:
I think this saveEntity is generic and can be elevated to BaseActor...

Reply:
we can do that , but as of now i am not able to see any business requirement to use this method in any other actor

Reply:
Understood that UserAttributeProcessorActor is in between User and AddressActor